fastai/data/core.py

```diff
@@ -188,7 +207,16 @@ def from_dsets(cls, *ds, path='.',  bs=64, device=None, dl_type=TfmdDL, **kwargs
         return cls(*[dl_type(d, bs=bs, **k) for d,k in zip(ds, kwargs)], path=path, device=device)
 
     @classmethod
-    def from_dblock(cls, dblock, source, path='.',  bs=64, val_bs=None, shuffle=True, device=None, **kwargs):
+    def from_dblock(cls,
+        dblock, # `DataBlock` object
+        source, # Source of data. Can be `Path` to files
+        path:(str, Path)='.', # Path to put `DataLoaders`
```

Comment:
```suggestion
        path:(str, Path)='.', # Path to put in `DataLoaders`
```
I think I would add the word `in` here because we don't actually store the dls at this location